### ISSUE TITLE
Parent Model

### ISSUE THREAD

**Issue by mattzollinhofer** [mattzollinhofer/teachy#8]

**Comment by wafflestomper** [comment]
Access to their child's record.  Would rather not have the teacher link the student/parent account, but somehow allow parent to request access to either student or teacher?

**Comment by mattzollinhofer** [comment]
Any more thoughts on what that interaction might look like? Parent requests to be able to see kids work, then kid or teacher can approve?

**Comment by wafflestomper** [comment]
That's what I was thinking.  It is hard to get parent's email address and get them involved, but it would be easy to tell them to request access if they were interested.

One way this would work (that I can see happening) is that a student emails their parents (which we require) and have them click on the link to the playbook - zollinhofer.com/math/smithj22.  If someone tries to go there without access, there is a screen that asks them to create an account (and asks who their kid is) and then they can be accepted.

Does that make sense?

**Comment by mattzollinhofer** [comment]
Yup, I'm into that. I'll update this issue as the parent model. But the interaction you described can be two different issues, can you create them? One the student initiating and one the parent initiating.

**Comment by wafflestomper** [comment]
What do you mean by "create" them?
